### PR TITLE
SLE-113 Repeated files analyzed

### PR DESCRIPTION
**Pull request by dbmeneses**

### CODE REVIEW

File: org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/jobs/AnalyzeProjectJob.java [hunk 1]
```diff
@@ -220,7 +223,11 @@ private void runAnalysisAndUpdateMarkers(final IProgressMonitor monitor, IProjec
           // Try to get a cached copy (for virtual file systems)
           localFile = fileStore.toLocalFile(EFS.CACHE, monitor);
         }
-        final Path filePath = localFile.toPath();
+        final Path filePath = localFile.toPath().toAbsolutePath();
+        if(uniqueFilePaths.contains(filePath.toString())) {
```
Comment (henryju) [inline org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/jobs/AnalyzeProjectJob.java]: Missing space

File: org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/jobs/AnalyzeProjectJob.java [hunk 2]
```diff
@@ -212,6 +213,8 @@ private void runAnalysisAndUpdateMarkers(final IProgressMonitor monitor, IProjec
     String allTestPattern = PreferencesUtils.getTestFileRegexps();
     String[] testPatterns = allTestPattern.split(",");
     final List<PathMatcher> pathMatchersForTests = createMatchersForTests(testPatterns);
+    Set<String> uniqueFilePaths = new HashSet<String>();
```
Comment (henryju) [inline org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/jobs/AnalyzeProjectJob.java]: Not using SonarLint :) ?

Comment (SonarTech) [inline org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/jobs/AnalyzeProjectJob.java]: MINOR Replace the type specification in this constructor call with the diamond operator ("<>"). [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2293)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Pull request closed without merging**